Apps/WebClient/src/ClientApp/src/views/reports.vue

```diff
@@ -207,254 +207,248 @@ export default class ReportsView extends Vue {
 
 <template>
```

Comment:
I think we should have a quick chat to explain the changes to this view and to the loading component.